packages/fast-usdc-contract/src/fast-usdc.contract.ts

```diff
@@ -193,9 +193,33 @@ export const contract = async (
   });
 
   const zoeTools = makeZoeTools(zcf, vowTools);
+  const { advanceFunds } = orchestrateAll(
+    // @ts-expect-error flow membrance type debt
+    { advanceFunds: flows.advanceFunds },
+    {
+      chainHub, // XXX excess authority?
+      feeConfig,
+      getNobleICA,
+      log: makeTracer('AdvanceFunds'),
+      settlementAccount: settleAccountV,
+      statusManager,
+      usdc: harden({
```

Comment:
Happy to see this absent in baggage, so we can change it later.

Reply:
What kind of change do you expect? An upgrade must redefine flows in a way that it continues to perform the same steps for active flows. Since an invocation captures arguments, a flow implementation could have different behavior based on the shape of arguments, but that seems unmaintainable. At that point it's likely easier to just define a "v2" flow if you're going to change the initial steps.